src/engine/controls/cuecontrol.h

```diff
@@ -98,18 +99,21 @@ class HotcueControl : public QObject {
     void setColor(mixxx::RgbColor::optional_t newColor);
     mixxx::RgbColor::optional_t getColor() const;
 
-    // Used for caching the preview state of this hotcue control.
+    // Used for caching the preview state of this hotcue control
+    // for the case the cue is deleted during preview.
     mixxx::CueType getPreviewingType() const {
-        return m_previewingType;
-    }
-    void setPreviewingType(mixxx::CueType type) {
-        m_previewingType = type;
+        return m_previewingType.getValue();
     }
     double getPreviewingPosition() const {
-        return m_previewingPosition;
+        return m_previewingPosition.getValue();
     }
-    void setPreviewingPosition(double position) {
-        m_previewingPosition = position;
+    void storePreviewingActivateData() {
```

Comment:
The name of the function does not tell me what it is actually doing. What is "activate data"?